sysctl.d/50-coredump.conf.in

```diff
@@ -9,4 +9,4 @@
 # and systemd-coredump(8) and core(5) for the explanation of the
 # setting below.
 
-kernel.core_pattern=|@rootlibexecdir@/systemd-coredump %P %s %t %c %h
+kernel.core_pattern=|@rootlibexecdir@/systemd-coredump %P %p %s %t %c %h
```

Comment:
not happy with this change either for compat reasons. I think it would be nice if we'd use the same pattern as before, but then use the usual SCM_CREDENTIALS trick for translating the pid/uid/gid from the host's data into the containers. i.e. before joining the child process into the container, let's enqueue a datagram on an AF_UNIX/SOCK_DGRAM socketpair() socket, then after joining the container read it from the socket, and the kernel will have translated all three fields for us, which we then can pass to the core pattern tool in the container.

Such an approach I think is more robust and doesn't create core_pattern compat issues

Reply:
See my comment below. I think we should minimize the number of translations rather than trying to handle them manually.

Reply:
Well, the keep virtualizing these ids (quota ids are next!) and thus they will keep adding more and more specifies and each time our cmdline changes. I'd really avoid that: let's use the same fixed command line we already use, but do the translation correctly if the data we want translated.